src/main/java/graphql/introspection/Introspection.java

```diff
@@ -327,6 +357,8 @@ private static String print(Object value, GraphQLInputType type) {
         register(__Type, "enumValues", enumValuesTypesFetcher);
         register(__Type, "inputFields", inputFieldsFetcher);
         register(__Type, "ofType", OfTypeFetcher);
+        register(__Type, "name", nameGraphQLElementDataFetcher);
+        register(__Type, "description", descriptionGraphQLElementDataFetcher);
```

Comment:
The other fetchers are just called "xxxFecther"  I would make it consistent

Reply:
changed it